dffml/df/base.py

```diff
@@ -175,6 +182,47 @@ def load(cls, loading=None):
         return loading_classes
 
 
+def create_definition(name, param_annotation):
+    if param_annotation in primitive_types:
+        return Definition(
+            name=name,
+            primitive=primitive_convert.get(
+                param_annotation, param_annotation.__name__
+            ),
+        )
+    elif (
```

Comment:
Let's add an `elif` block to check if `get_origin(param_annotation)` is `Optional`, if it is, `return create_definition(name, list(get_args(param_annotation))[0])`

Reply:
sure, thanks